content/influxdb/v2.4/write-data/replication/replicate-data.md

```diff
@@ -16,16 +16,26 @@ related:
 ---
 
 Use InfluxDB replication streams (InfluxDB Edge Data Replication) to replicate
-the incoming data of select buckets to one or more buckets on a remote
-{{% oss-only %}}InfluxDB OSS, InfluxDB Cloud, or InfluxDB Enterprise instance.{{% /oss-only %}}
-{{% cloud-only %}}InfluxDB Cloud instance.{{% /cloud-only %}}
+the incoming data of select buckets to one or more buckets on a remote 
+InfluxDB OSS, InfluxDB Cloud, or InfluxDB Enterprise instance.
+
+The following guide demonstrates how to replicate data from InfluxDB OSS to 
+InfluxDB Cloud and can also be used to replicate data to the following instances:
+- InfluxDB OSS to InfluxDB OSS
+- InfluxDB OSS to InfluxDB Cloud
+- InfluxDB OSS to InfluxDB Enterprise
```

Comment:
```suggestion
- InfluxDB OSS to InfluxDB OSS
- InfluxDB OSS to InfluxDB Enterprise
```

or something like this, b/c you already have the the note for OSS and Enterprise:

The following guide shows how to replicate data for the following use cases:
- InfluxDB OSS to InfluxDB OSS
- InfluxDB OSS to InfluxDB Cloud
- InfluxDB OSS to InfluxDB Enterprise
...
Configure remote connection for the InfluxDB instance that you want to replicate data to.